Makefile

```
@@ -74,6 +77,23 @@ down: ## remove stack (warning: it removes the database container)
 	@$(COMPOSE) down || echo WARNING: unable to remove the stack. Try to stop linked containers or networks first.
 .PHONY: down
 
+fixtures: 
+	@$(COMPOSE) up -d elasticsearch
+	@echo "Wait for database to be up..."
+	@$(WAIT_ES)
+	zcat ./fixtures/elasticsearch/lrs.json.gz | \
+	  python3 ./scripts/elasticsearch.py | \
+	  sed 's/timestamp/@timestamp/g' | \
+	  $(COMPOSE_RUN) ralph \
+		  "bash" \
+		  "-c" \
+		  "ralph push -b es --es-index statements-fixtures --es-hosts http://elasticsearch:9200"
```

It's already set in the environment.

```suggestion
		  "ralph push -b es --es-index statements-fixtures"
```